scipy/special/_add_newdocs.py

```diff
@@ -10589,8 +10589,34 @@ def add_newdoc(name, doc):
 
     See Also
     --------
-    erf, erfc, scipy.stats.norm, log_ndtr
+    log_ndtr: Logarithm of ndtr
+    erf: Error function
+    erfc: 1 - erf
+    scipy.stats.norm: Normal distribution
```

Comment:
This is to comply with numpydoc
```suggestion
    log_ndtr : Logarithm of ndtr
    erf : Error function
    erfc : 1 - erf
    scipy.stats.norm : Normal distribution
```